src/components/ClickableStyle/ClickableStyle.module.css

```diff
@@ -454,21 +454,25 @@
 
 /**
  * Link button styles
+ * 1) Override default Clickable styles
+ * 2) Smaller gap between text and icon in link button style
  */
 .clickable-style--link {
   @mixin textLink;
 
-  border-radius: 0;
+  /* 1 */
   border: 0;
+  border-radius: 0;
   background-color: transparent;
+
+  gap: 0.25em; /* 2 */
```

Comment:
I moved this from the mixin because it will only have an effect here due to the `display: inline-flex` base style.

Reply:
I'm also wondering if I should instead put `display: inline` on the mixin and move this to the `Button` styles. I feel like the gap behavior should be the default but 🤷